clusters/app.ci/assets/bugzilla-backporter.yaml

```diff
@@ -0,0 +1,111 @@
+kind: List
+apiVersion: v1
+items:
+- kind: ImageStream
+  apiVersion: image.openshift.io/v1
+  metadata:
+    name: bugzilla-backporter
+    namespace: ci
+  spec:
+    lookupPolicy:
+      local: true
+    tags:
+    - name: latest
+      importPolicy:
+        scheduled: true
+      from:
+        kind: DockerImage
+        name: registry.svc.ci.openshift.org/ci/bugzilla-backporter
+- apiVersion: v1
+  kind: Service
+  metadata:
+    namespace: ci
+    labels:
+      app: prow
+      component: bugzilla-backporter
+    name: bugzilla-backporter
+  spec:
+    ports:
+    - name: main
+      port: 80
+      protocol: TCP
+      targetPort: main
+    - name: metrics
+      port: 9090
+      protocol: TCP
+      targetPort: metrics
+    selector:
+      app: prow
+      component: bugzilla-backporter
+- apiVersion: route.openshift.io/v1
+  kind: Route
+  metadata:
+    labels:
+      app: prow
+      component: bugzilla-backporter
+    name: bugzilla-backporter
+    namespace: ci
+  spec:
+    port:
+      targetPort: main
+    to:
+      kind: Service
+      name: bugzilla-backporter
+- apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    namespace: ci
+    name: bugzilla-backporter
+    annotations:
+      image.openshift.io/triggers: '[{"from":{"kind":"ImageStreamTag","name":"bugzilla-backporter:latest"},"fieldPath":"spec.template.spec.containers[?(@.name==\"bugzilla-backporter\")].image"}]'
+  spec:
+    replicas: 1
+    strategy:
+      type: RollingUpdate
+    selector:
+      matchLabels:
+        app: prow
+        component: bugzilla-backporter
+    template:
+      metadata:
+        labels:
+          app: prow
+          component: bugzilla-backporter
+      spec:
+        containers:
+        - name: bugzilla-backporter
+          image: bugzilla-backporter:latest
+          args:
+          - -bugzilla-endpoint=https://bugzilla.redhat.com/
+          - -bugzilla-api-key-path=/etc/bugzilla/api
+          - -log-level=debug
+          ports:
+          - name: main
+            containerPort: 8080
+          - name: metrics
+            containerPort: 9090
+          livenessProbe:
+            httpGet:
+              path: /healthz
+              port: 8081
+            initialDelaySeconds: 3
+            periodSeconds: 3
+          readinessProbe:
+            httpGet:
+              path: /healthz/ready
+              port: 8081
+            initialDelaySeconds: 10
+            periodSeconds: 3
+            timeoutSeconds: 600
+          volumeMounts:
+          - name: config
+            mountPath: /etc/config
+            readOnly: true
+          resources:
+            requests:
+              memory: "2Gi"
+              cpu: "200m"
+        volumes:
+        - name: config
+          configMap:
+            name: config
```

Comment:
I wanted to leave a volume mount for the secret file here. I am not really sure if that gets configured here or someplace else. Left this in as a placeholder till then